tests/unit/utils/test_openstack.py

```diff
@@ -289,7 +282,6 @@ def test_openstack_release_setter():
 def test_openstack_release_setter_by_date():
     openstack_release = OpenStackRelease("2023.1")
     assert openstack_release.codename == "antelope"
-    assert openstack_release.next_release == "bobcat"
```

Comment:
Can we assert that the next release is caracal here?

Reply:
Thanks, it's now tested in `test_plan.py`